Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.py

```diff
@@ -1704,11 +1981,16 @@ def main():
             return_results(reset_auth())
         elif demisto.command() == 'msg-generate-login-url':
             return_results(generate_login_url(client.ms_client))
+        elif demisto.command() == 'msg-advanced-hunting':
+            return_results(advanced_hunting_command(client, args))
+        elif demisto.command() == 'msg-list-security-incident':
+            return_results(get_list_security_incident_command(client, args))
+
```

Comment:
where is the new third command here in the main function?

Reply:
You're right, I fixed it